<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlagsEditor</class>
 <widget class="QWidget" name="FlagsEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>107</width>
    <height>24</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FlagsEditor</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="ClickableLabel" name="labelC">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>c</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ClickableLabel" name="labelP">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>p</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ClickableLabel" name="labelA">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>a</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ClickableLabel" name="labelZ">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>z</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ClickableLabel" name="labelS">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>s</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ClickableLabel" name="labelT">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>t</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ClickableLabel" name="labelI">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>i</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ClickableLabel" name="labelD">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>d</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ClickableLabel" name="labelO">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>o</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>ClickableLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
